sklearn/cluster/tests/test_k_means.py

```diff
@@ -692,7 +702,7 @@ def test_integer_input(Estimator, array_constr, dtype, init):
     assert km.cluster_centers_.dtype == np.float64
 
     expected_labels = [0, 1, 1, 0, 0, 1]
-    assert_array_equal(km.labels_, expected_labels)
+    assert v_measure_score(km.labels_, expected_labels) == 1.0
```

Comment:
to avoid rounding errors because `v_measure_score` outputs a float
```suggestion
    assert_allclose(v_measure_score(km.labels_, expected_labels), 1.0)
```